packages/affiliates/payouts/devmining/2021-04-19_2021-04-26_0024.json

```diff
@@ -6,41 +6,41 @@
-      ["0xf215778F3a5e7Ab6A832e71d87267Dd9a9aB0037","0xb8E237CC8D2E81b7C9C9C4B5877200f85100fC87","1.2.2"],
-      ["0x7c4090170aeADD54B1a0DbAC2C8D08719220A435","0x7eb1e51351380E67F641B84545bf41eCdc10298E","1.2.2"],
-      ["0x9E929a85282fB0555C19Ed70942B952827Ca4B0B","0x1Da788aBB922325E0ea2533019bf09f8a7525AC0","2.0.1"],
-      ["0x6DA66C15823cFf681DaD6963fBD325a520362958","0x1Da788aBB922325E0ea2533019bf09f8a7525AC0","2.0.1"],
-      ["0xD50fbace72352C2e15E0986b8Ad2599627B5c340","0x9C8F094985d12d80e930d2abBB6471EDF68d3bEB","1.2.2"],
-      ["0x45788a369f3083c02b942aEa02DBa25C466a773F","0xDAa953f01048253A201BA9B0bd0786575f9C2468","2.0.1"],
-      ["0x0388f65C185a7E7D857BB142185381d97a4bc747","0x634ad76076B79c4d5aFbA2398f2bE346D5aC4d1c","2.0.1"],
-      ["0x9bB1f39b6DB45BD087046385a43EAb7b60C52e7D","0x634ad76076B79c4d5aFbA2398f2bE346D5aC4d1c","2.0.1"],
-      ["0x14A415Dd90B63c791C5dc544594605c8bC13Bc8D","0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E","2.0.1"],
-      ["0x1066E9D2E372d01A0F57bB6f231D34Ce4CEd228e","0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E","2.0.1"],
-      ["0xa24Ba528Be99024f7F7C227b55cBb265ecf0C078","0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E","2.0.1"],
-      ["0xd60139B287De1408f8388f5f57fC114Fb4B03328","0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E","2.0.1"],
-      ["0x8E51Ad4EeB19693751a9A3E36b8F098D891Ddc7f","0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E","2.0.1"],
-      ["0x4E8d60A785c2636A63c5Bd47C7050d21266c8B43","0x97990B693835da58A281636296D2Bf02787DEa17","2.0.1"],
-      ["0x4F1424Cef6AcE40c0ae4fc64d74B734f1eAF153C","0x97990B693835da58A281636296D2Bf02787DEa17","2.0.1"],
-      ["0x964Be01cCe200e168c4ba960a764cBEBa8C01200","0x9a9dcd6b52B45a78CD13b395723c245dAbFbAb71","2.0.1"],
-      ["0xDB2E7F6655de37822c3020a8988351CC76caDAD5","0xAd2B08F878C0Be4C4162A72bbF7c0f547F93bD9f","2.0.1"],
-      ["0x46f5E363e69798a74c8422BFb9EDB63e3FB0f08a","0x2CA781A4dcA1f37F52696238e70A004746068F3b","2.0.1"],
-      ["0xb40BA94747c59d076B3c189E3A031547492013da","0xd332f9Faf23Ba97d9f0f9EE9b3d8798C49Ff269B","2.0.1"],
-      ["0x312Ecf2854f73a3Ff616e3CDBC05E2Ff6A98d1f0","0xFc41Ad4FbAa1e6Ed9690E989121122fF6755C918","2.0.1"],
-      ["0x56BaBEcb3dCaC063697fE38AB745c10181c56fA6","0x7eb1e51351380E67F641B84545bf41eCdc10298E","2.0.1"],
-      ["0x10E018C01792705BefB7A757628C2947E38B9426","0xDAa953f01048253A201BA9B0bd0786575f9C2468","2.0.1"],
-      ["0xC9E6C106C65eDD67C83CC6e3bCd18bf8d2Ebf182","0x45Ea614a7Ea47Ec393BDA310F901702DB1347df9","2.0.1"],
-      ["0xCef85b352CCD7a446d94AEeeA02dD11622289954","0xb8E237CC8D2E81b7C9C9C4B5877200f85100fC87","2.0.1"],
-      ["0xd9af2d7E4cF86aAfBCf688a47Bd6b95Da9F7c838","0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35","2.0.1"],
-      ["0x0f4e2a456aAfc0068a0718E3107B88d2e8f2bfEF","0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35","2.0.1"]
+      ["0xaB3Aa2768Ba6c5876B2552a6F9b70E54aa256175", "0x45Ea614a7Ea47Ec393BDA310F901702DB1347df9", "1.2.2"],
```

Comment:
@daywiss @evaldofelipe Why are these files not formatted correctly? And how is this passing our ci lint job?

Reply:
The rewards linter was added very recently. The next files will not have this issue.